<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QMainWindow" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>749</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Todo List Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="TreeWidgetAcceptSelf" name="tree_view">
      <property name="dragDropMode">
       <enum>QAbstractItemView::InternalMove</enum>
      </property>
      <property name="defaultDropAction">
       <enum>Qt::MoveAction</enum>
      </property>
      <column>
       <property name="text">
        <string notr="true">Title</string>
       </property>
      </column>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLineEdit" name="text_new">
        <property name="placeholderText">
         <string>New Item</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="combo_position">
        <item>
         <property name="text">
          <string>Below current</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Above current</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Child of current</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Top</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Bottom</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_add">
        <property name="text">
         <string>Add</string>
        </property>
        <property name="icon">
         <iconset theme="list-add">
          <normaloff/>
         </iconset>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>749</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="action_import"/>
    <addaction name="action_export"/>
    <addaction name="separator"/>
    <addaction name="action_quit"/>
   </widget>
   <addaction name="menu_File"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="action_import">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="document-import"/>
   </property>
   <property name="text">
    <string>&amp;Import...</string>
   </property>
  </action>
  <action name="action_export">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="document-export"/>
   </property>
   <property name="text">
    <string>&amp;Export...</string>
   </property>
  </action>
  <action name="action_quit">
   <property name="icon">
    <iconset theme="application-exit"/>
   </property>
   <property name="text">
    <string>&amp;Quit</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TreeWidgetAcceptSelf</class>
   <extends>QTreeWidget</extends>
   <header>tree_widget_accept_self.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>text_new</sender>
   <signal>returnPressed()</signal>
   <receiver>button_add</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>550</y>
    </hint>
    <hint type="destinationlabel">
     <x>697</x>
     <y>560</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>action_quit</sender>
   <signal>triggered()</signal>
   <receiver>Window</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
